habitat/datasets/eqa/mp3d_eqa_dataset.py

```diff
@@ -56,6 +56,9 @@ def __init__(self, config: Config = None) -> None:
         ) as f:
             self.from_json(f.read())
 
+        if config.NUM_EPISODE_SAMPLE >= 0:
```

Comment:
Let's move it to `sample_episodes(self, num_episodes: int)` to have this check for all calls.

Reply:
@JasonJiazhiZhang this checks not needed anymore.